src/index.js

```diff
@@ -76,7 +73,7 @@ export default class IndexedCache {
   }
 
   // Initialize the indexedDB database and create the store.
-  async _initDB (dbName, storeName) {
+  _initDB (dbName, storeName) {
```

Comment:
We `await` on `_initDB` to wait for the DB to be initialized.

Reply:
I think `async` is only needed when we use `await` inside the function.
We are using `await` on `_initDB` as it returns a `Promise` not because it's `async`.

Reply:
> We `await` on `_initDB` to wait for the DB to be initialized.

As `__initDB` is explicitly returning a `promise` , so we can use `await` on it, even if function is not marked as `async`